Content.Server/Fax/FaxSystem.cs

```diff
@@ -300,6 +303,13 @@ private void OnToggleInterface(EntityUid uid, FaxMachineComponent component, Aft
         UpdateUserInterface(uid, component);
     }
 
+    private void OnFileButtonPressed(EntityUid uid, FaxMachineComponent component, FaxFileMessage args)
+    {
+        args.Content = args.Content.Substring(0, Math.Min(args.Content.Length, 10000));
+        args.Name = args.Name.Substring(0, Math.Min(args.Name.Length, 64));
+        PrintFile(uid, component, args);
+    }
```

Comment:
As elsewhere put 10000 somewhere shared so they reference the same constant and remove Name.